server/services/storage/agent_framework_storage_service.ts

```diff
@@ -146,109 +146,93 @@ export class AgentFrameworkStorageService implements StorageService {
   }
 
   async deleteConversation(conversationId: string): Promise<ConversationOptResponse> {
-    try {
-      const response = await this.client.transport.request({
-        method: 'DELETE',
-        path: `${ML_COMMONS_BASE_API}/memory/${encodeURIComponent(conversationId)}`,
-      });
-      if (response.statusCode === 200) {
-        return {
-          success: true,
-        };
-      } else {
-        return {
-          success: false,
-          statusCode: response.statusCode,
-          message: JSON.stringify(response.body),
-        };
-      }
-    } catch (error) {
-      throw new Error('delete conversation failed, reason:' + JSON.stringify(error.meta?.body));
```

Comment:
Shall we not throw error here just return 
```ts
      return {
        success: false,
        message: "delete conversation failed",
      };
```
I think we still need to catch error from `this.client.transport.request`